src/services/rates/currencyConverter.test.ts

```
@@ -0,0 +1,312 @@
+      (prisma.acbuRate.findFirst as jest.Mock).mockResolvedValue(mockRate);
+
+      const currencies = ["NGN", "KES", "ZAR", "EGP", "GHS", "RWF", "XOF", "MAD", "TZS", "UGX", "EUR", "GBP", "USD"];
+
+      for (const currency of currencies) {
+        const result = await convertLocalToUsd(1000, currency);
+        expect(typeof result).toBe("number");
+        expect(result).toBeGreaterThan(0);
+      }
+    });
+
+    it("should preserve decimal precision with high-value conversions", async () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)